upath/registry.py

```diff
@@ -46,14 +50,14 @@ def __getitem__(self, item: str) -> type[PT] | None:
 
 
 @lru_cache()
-def get_upath_class(protocol: str) -> type[PT] | None:
-    """return the upath cls for the given protocol"""
+def get_upath_class(protocol: str) -> type[PT] | type[Path] | None:
+    """Return the upath cls for the given protocol."""
     cls: type[PT] | None = _registry[protocol]
     if cls is not None:
         return cls
     else:
         if not protocol:
-            return None  # we want to use pathlib for `None` protocol
+            return Path  # we want to use pathlib for `None` protocol
```

Comment:
I'm not sure if we should change `get_upath_class` behavior for now.
We should figure out a solution for #90 before introducing a change here.